widget/table_test.go

```diff
@@ -303,3 +303,32 @@ func TestTable_ShowVisible(t *testing.T) {
 	cellRenderer.Refresh()
 	assert.Equal(t, 8, len(cellRenderer.Objects()))
 }
+
+func TestTable_SeparatorThicknessZero_NotPanics(t *testing.T) {
+	test.NewApp()
+	defer test.NewApp()
+
+	test.ApplyTheme(t, &separatorThicknessZeroTheme{theme.DefaultTheme()})
+
+	table := NewTable(
+		func() (int, int) { return 500, 150 },
+		func() fyne.CanvasObject {
+			return NewLabel("placeholder")
+		},
+		func(TableCellID, fyne.CanvasObject) {})
+
+	assert.NotPanics(t, func() {
+		table.Resize(fyne.NewSize(400, 644))
+	})
+}
+
+type separatorThicknessZeroTheme struct {
+	fyne.Theme
+}
+
+func (separatorThicknessZeroTheme) Size(s fyne.ThemeSizeName) float32 {
+	if s == theme.SizeNameSeparatorThickness {
+		return 0
+	}
+	return theme.DefaultTheme().Size(s)
```

Comment:
If you delegate to the embedded theme instead you'll get the correct fallback instead of accessing a new theme.

Reply:
Oh yes, good catch. I only removed the extra code and didn't realize it, will update! :)

Reply:
done